### DISCUSSION TITLE
What are your approaches for modeling oral absorption with late Tmax?

### DISCUSSION THREAD

**Discussion by prvmalik** [Open-Systems-Pharmacology/Forum#942] [Questions & Problems]
Delayed Tmax is something PBPK modeling has a problem with. It could be because the compound has poor solubility, is a substrate for a unknown intestinal transporter, is subject to intestinal wall metabolism, and a variety of other reasons. I typically trial a series of approaches when faced with this scenario and I can't say one works reliably - sometimes none of them work! But I think there are others out there. I thought we could share our methods for tackling delayed Tmax. I often work in this order:

1. Try an empirical weibull function dissolution profile. I optimize spec intestinal permeability, 50% diss time +/- lag time. I find that diss shape is not uniquely identifiable when optimized together with spec intestinal permeability. If I can use a Caco-2 permeability as a fixed parameter, then I can fix spec intestinal permeability and optimize 50% diss time, diss shape +/- lag time. Of course, run optimizations at least x6 from randomized start values to confirm identifiability
2. Optimizing the surface area enhancement factors (x5) for the small intestine mucosal compartments
3.  If there is evidence of an intestinal transporter or segment-dependent absorption, add an apical influx transporter and basolateral efflux transporter on the proposed segments (e.g. upper and lower ileum). I assume that the influx transporter is rate limiting to absorption and set basolat transport to be fast (Km = 5000 uM and Vmax = 5000 umol/min/L or whatever the units are). Set drug to be dissolved, optimize spec intestinal permeability, influx transporter Vmax + Km if there is dose-dependent absorption or set Km = 5000 uM to have linear absorption and optimize Vmax.
4. Optimize solubility in intestinal segments, last resort, rarely works or is relevant.

Let me know if you have good/reliable approaches for handling this, that maintain some sort of physiologic relevance. It's the biggest challenge in PBPK modeling for me at the moment!

**Comment by tobiasK2001** [comment]
Dear Paul,

thanks for raising this interesting point. In general I can agree with your workflow. Depending on the compounds I might have some different experience. In my cases the delayed Tmax seems to come most often due to solubility issues (supersaturation, reprecipitation and redissolution. e.g for typical lipophilic BCS class 2 compounds with basic PKA).  I don#t have so much experience with cases were I could assume high intestinal gut wall metabolism or unknown intestinal transportes. 
I used for example this workflow: 
1. set disposition with iv data from absolute F study (if available)
2. optimize spec. intestinal permeability with po data from solution formulation (if available) or use Caco-2 permeability fixed if it describes in vivo data
3. Try empirical weibull function dissolution profile were 50% diss time is fitted per each dose level and shape is fitted over all doses. 
4. Depending on the curve shape / Tmax fit enhance solubility in intestinal segment. 

I also tried a more mechanistic approaches with particle dissolution, but that was not always successful and for some compounds I had the feeling the OSP-suite is a bit limited. E.g. I had a compound with a similar profile like aprepitant describe by [Pepin et 2021](https://doi.org/10.1208/s12248-020-00548-8) with a "bump" around 24 hours (Fig.6). I thought I could use a somewhat similar approach like the authors describe in the supplement. I tried to split the dose in a "liquid" fast absorption part and a "colonic absorbed part" with using a "Advanced protocol" were 50% of the dose is administered oral and the other 50% is administered in the large intestinal lumen (as particle or dissolution function). That should kind of mimic the absorption coming from redissolution of precipitated parts of the drug.  But technically it seems not to be possible to create this kind of advanced protocols. Maybe there's some way and I could not find it. But in the end I could at least empirically describe the data with an empirical weibull function and assuming enhanced colonic solubility.
Generally it seems that the dissolution, precipitation and redissolutions processes in the intestine are quite complex and it is  difficult to explain luminal absorption mechanistically even for compounds with wealth of data (e.g. Itraconazole as described by very interesting review of [ [Boyed et al 2019](https://www.sciencedirect.com/science/article/pii/S0928098719302301?via%3Dihub)]).
So I guess it will still remain a huge challenge.

Hope that might helps nevertheless.

Best, Tobias

**Reply by prvmalik** [reply]
Great scientific content Tobias. Thank you for linking those papers. 

It's good to know about your experience with the particle dissolution model as trialing it takes a significant investment of time and parameterization.

I have tried setting up 'advanced' dissolution protocols similar to the apprepitant scenario that you are describing. I use the "Table" option dissolution and enter values manually. Of course this is very crude. If you wanted 50% absorbed quickly and 50% absorbed in the large intestine, you could have the first 50% of the dissolution (fraction dissolved 0 - 0.5) occur within 0-15 minutes, hold it constant at 0.5 for 4 hours and then deposit the rest of the formulation (fraction dissolved 0.5 to 1.0) in the large intestine. Of course this is only a last resort approach when the mechanisms are not well understood.

In my reading of the midazolam model in the library I saw a fantastic approach by @sfrechen (I think?). The P intracellular -> interstitial and P interstitial -> intracellular for each mucosal segment can be optimized to slow transit from mucosa to the liver. This can have the effect of delaying Tmax or improving bioavailability estimates by increasing gut wall metabolism via residence time in the mucosa. All in all it is a group of 22 parameters per simulation that must be entered into the parameter identification tool but they are all equal to the same value.

**Comment by StephanSchaller** [comment]
Hi all, thanks for the very interesting Discussion.

Yes, a multi-phase release could be modeled with the table formula release or hopefully the double-Weibull in the future (a current feature request: https://github.com/Open-Systems-Pharmacology/PK-Sim/issues/1701). A phenomenon like the double-peak absorption which is sometimes observed could also be empirically described by this (although the phenomenon might be due to a phasic gastric emptying or efflux transporter expression along the gut wall pushing absorption to a lower gut segment).
Late 24h peaks could also be due to EHC involving conjugation (liver --> bile) and deconjugation (large intestine) followed by re-absorption.
Further factors to influence Tmax are of course the gastric emptying and gut transit times which might be affected by some drugs.

Cheers,
Stephan

**Comment by Ricardo-DO** [comment]
Dear all,

Thanks for raising the point of the late tmax modelling, very interesting points from all of you.

If we consider a highly lipophilic basic compound, a non-specific gut binding could be taking place (e.g. lysosomal trapping).
How can this add be added to the model?
Could it be by adding a protein expressed in the gut wall and optimise the kd and kdiss to match the tmax?

Thanks,
Ricardo

**Reply by prvmalik** [reply]
Interesting concept. If lysosomal trapping you might be better off working with the basolateral intestinal cell permeability parameter as proposed by @sfrechen . For a protein binding partner for sure you could insert one in the gut mucosa and optimize KD and koff.
